src/main/c/src/SerialImp.c

```diff
@@ -306,9 +306,9 @@ JNIEXPORT void JNICALL RXTXPort(Initialize)(
 	}
 	if(strcmp(name.release,UTS_RELEASE)!=0)
 	{
-//		sprintf( message, LINUX_KERNEL_VERSION_ERROR, UTS_RELEASE,
+//		snprintf( message, sizeof(message), LINUX_KERNEL_VERSION_ERROR, UTS_RELEASE,
 //			name.release );
-		report( message );
```

Comment:
Yikes, was `report` really being called here even though `message` was uninitialized? 😬 Thanks for fixing.

Reply:
Yes, I reckoned this was not intended ;) After looking at the code again I guess we should re-introduce this because it is followed by a `getchar()`. Without any message this will be confusing. I think I know why it was commented: the message `LINUX_KERNEL_VERSION_ERROR` is very long (nearly 400 characters) and did not fit into the original buffer. Buffer size is now 512 and I re-introduced the message. Do you agree with this approach?

Reply:
I'd remove the whole ifdef block from L290 to L315. Seems like it is broken for at least 7 years and nobody noticed. Also, I don't think the kernel version is that important, that you need to print a warning and demand a key press in a library like this 🙂

Reply:
Thanks for the hint 👍  I removed both #if blocks relating to Linux system name handling now.